owncloudApp/src/main/java/com/owncloud/android/utils/UriUtils.java

```diff
@@ -226,13 +230,13 @@ private static String getDisplayNameFromContentResolver(Uri uri, Context context
         String mimeType = context.getContentResolver().getType(uri);
         if (mimeType != null) {
```

Comment:
You could for example export this if clause.

Reply:
Sure, but it is not the purpose of this PR, i just moved some hardcoded strings into constants. Probably we will convert to kotlin and refactor this file in upcoming PRs.